orgparse/__init__.py

```diff
@@ -138,25 +139,27 @@ def load(path):
     else:
         orgfile = path
         filename = path.name if hasattr(path, 'name') else '<file-like>'
-    return loadi((l.rstrip('\n') for l in orgfile.readlines()),
-                 filename=filename)
 
+    if not env:
+        env = OrgEnv(filename=filename)
```

Comment:
I had in mind doing this bit in `parse_lines` (so it would have signature `def parse_lines(lines, filename, env=None)`. That way it would be handled in one place and will be backwards compatible (someone might have been using `parse_lines`) before. What do you think?

Reply:
P.S. also meant that `loadi`/`loads` etc would also have extra `env=None` argument that they will simply pass down as `env=env`

Reply:
I tested a few signatures but wasn't convinced. In your example, something like `parse_lines(my_lines, "filename1", OrgEnv(filename="filename2"))` may lead to unexpected behavior.

But i totally agree, it's not a good idea to break the API for this.

Reply:
Good point too about filenames, but hopefully ignoring `filename` argument is not a big problem if the user is passing `OrgEnv` down (presumably they know what they are doing). Also when there is no proper filename it's `<undefined>/<lines>/<string>` anyway, so a placeholder only used for debugging.